apps/api/src/app/layouts/layouts.controller.ts

```diff
@@ -163,7 +163,7 @@ export class LayoutsController {
     );
   }
 
-  @Patch('/:layoutId')
+  @Put('/:layoutId')
```

Comment:
We are not sending the full entity when updating so `PATCH` was the right method. Could you rollback this change?